src/main/java/controller/UserController.java

```diff
@@ -3,33 +3,53 @@
 import annotation.RequestMapping;
 import model.HttpRequest;
 import model.HttpResponse;
+import model.User;
 import service.UserService;
+import session.SessionManager;
 import type.RequestMethod;
+
 import java.util.Map;
+import java.util.Optional;
 
 @RequestMapping(path = "/user")
-public class UserController implements Controller{
+public class UserController implements Controller {
 
     private final UserService userService = new UserService();
+    private SessionManager sessionManager;
+
+    public UserController(SessionManager sessionManager) {
+        this.sessionManager = sessionManager;
+    }
 
     @RequestMapping(path = "/create", method = RequestMethod.POST)
-    public void createAsPost(HttpRequest httpRequest, HttpResponse httpResponse) throws Exception {
+    public String create(HttpRequest httpRequest, HttpResponse httpResponse) throws Exception {
         String requestUrl = httpRequest.getPathInfo();
         Map<String, String> requestParams = httpRequest.getRequestBody();
         userService.join(requestParams);
-        httpResponse.redirect("/");
+        httpResponse.sendRedirect("/");
+        httpResponse.setStatus(HttpResponse.SC_SEE_OTHER);
+        return null;
     }
-    @RequestMapping(path = "/create", method = RequestMethod.GET)
-    public void createAsGet(HttpRequest httpRequest, HttpResponse httpResponse) throws Exception {
-        String requestUrl = httpRequest.getPathInfo();
-        String requestParams = httpRequest.getQueryString();
-        userService.join(requestParams);
-        httpResponse.redirect("/");
+
+    @RequestMapping(path = "/login", method = RequestMethod.POST)
+    public String login(HttpRequest httpRequest, HttpResponse httpResponse) {
+        Map<String, String> requestBody = httpRequest.getRequestBody();
+        String userId = requestBody.get("userId");
```

Comment:
```suggestion
        String userId = httpRequest.getParameter("userId");
```
map 을 받아서 처리하는 것보다 직접 바디의 parameter로 값을 리턴해 주는 메소드가 Request에 있으면 좋겠네요.